components/mag3110/include/mag3110.h

```diff
@@ -101,6 +101,18 @@ void mag3110_delete(mag3110_handle_t sensor);
  */
 esp_err_t mag3110_start(mag3110_handle_t sensor, const mag3110_data_rate_t data_rate);
 
+/**
+ * @brief Start mag3110 sensor to measure, without previous sensor calibration
+ *
+ * @param sensor object handle of mag3110
+ * @param[in] data_rate Data rate and oversampling settings
+ *
+ * @return
+ *     - ESP_OK Success
+ *     - ESP_FAIL Fail
+ */
+esp_err_t mag3110_start_no_calibration(mag3110_handle_t sensor, const mag3110_data_rate_t data_rate);
```

Comment:
```suggestion
esp_err_t mag3110_start_raw(mag3110_handle_t sensor, const mag3110_data_rate_t data_rate);
```
IMO, the description and name is not very clear. According to the datasheet 

> 0: Normal mode: data values are corrected by the user offset register values.
1: Raw mode: data values are not corrected by the user offset register values. 

So we should state here that the offset registers (that are set in `mag3110_calibrate` function) are ignore in raw mode

Reply:
changed to `mag3110_start_raw` as suggested